template-letter.tex

```diff
@@ -59,6 +59,7 @@
 $endif$
 \usepackage[unicode=true]{hyperref}
 $if(colorlinks)$
+\usepackage[dvipsnames]{xcolor}
```

Comment:
Can we drop this?  I try to stay as close as possible to the original template and it appears jgm [uses the color package because it's already bundled with hyperef](https://github.com/jgm/pandoc-templates/pull/107).